ansys/dpf/core/plotter.py

```diff
@@ -87,7 +87,7 @@ def plot_chart(self, fields_container):
         substr = fields_container[0].name.split("_")
         pyplot.ylabel(substr[0] + fieldMin.unit)
         pyplot.title(substr[0] + ": min/max values over time")
-        return pyplot.legend()
+        return pyplot.show()
```

Comment:
Actually, we should refactor this such that we create a `pyplot.figure` and then return the handle to that.  Sometimes (especially with unit testing), having figures pop-up isn't ideal.  If we pass the handle of the figure the user will be able to show the figure at will, and even adjust its size rather than using `pyplot.gcf`

Reply:
True, this would be an issue for batch-mode. How about adding a boolean argument to the function which would return the figure handle when set `True`, default value can be `False` which will return `pyplot.show()` and the chart pops-up?

For unit-testing the boolean can be set to `True`? Or when the user wants direct control on the figure handle. This would only add a functionality to the function and the original idea of the function remains intact. As the user expects from the function name `plot_chart` a pop-up 2D chart analogous to `plot_contour`.

Reply:
> True, this would be an issue for batch-mode. How about adding a boolean argument to the function which would return the figure handle when set `True`, default value can be `False` which will return `pyplot.show()` and the chart pops-up?

Works for me, let's just use `show=True` and then set it to `False` in our unit testing.  Otherwise, let's have it return the figure handle.

Reply:
`show=True` sounds good, I will make the necessary changes, thanks!